gc/gc-repository-jdbc/src/main/java/org/projectnessie/gc/contents/jdbc/SqlDmlDdl.java

```diff
@@ -169,13 +170,12 @@ private SqlDmlDdl() {}
           + "    FROM gc_live_set_contents \n"
           + "    WHERE live_set_id = ? AND content_id = ?";
 
-  static final List<String> ALL_CREATES =
-      Collections.unmodifiableList(
-          Arrays.asList(
-              CREATE_LIVE_SETS,
-              CREATE_LIVE_SET_CONTENTS,
-              CREATE_LIVE_SET_LOCATIONS,
-              CREATE_FILE_DELETIONS));
+  static final Map<String, String> ALL_CREATES =
+      Map.of(
```

Comment:
Should this maybe be a sorted map, so the output is rather deterministic for humans?